test/no-empty-alt-text.test.js

```diff
@@ -19,6 +19,8 @@ describe("GH003: No Empty Alt Text", () => {
         '<img alt="" src="https://user-images.githubusercontent.com/abcdef.png">',
         "<img alt='' src='https://user-images.githubusercontent.com/abcdef.png'>",
         '<img src="cat.png" alt="" /> <img src="dog.png" alt="" />',
+        '<img src="dog.png" />',
+        '<img alt src="dog.png" />',
```

Comment:
```suggestion
        '<img alt src="dog.png" />',
        '<img alt><img alt>'
```
I suspect the current code might not cover this usecase 🤔

Maybe if line 30 became:

```js
const matches = [...line.matchAll(htmlEmptyAltRegex), ...line.matchAll(htmlAltRegex)]
```